recipes/xs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: xs
+  version: 1.0.0
+
+source:
+  url: https://github.com/pratas/xs/archive/v1.tar.gz
+  md5: 9fd0d370bcdc3886858f775ce593d65f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - XS -n 1 /dev/null
+
+about:
+  home: https://github.com/pratas/xs
+  license: GPL-2.0
```

Comment:
We need a license file for GPL. Could you please add one to the repo I guess or if you have upstream power also to upstream :)

Reply:
There isn't a license file in the repo, but there is this: https://github.com/pratas/xs/blob/master/README.md#license. Sadly I don't have upstream power

Reply:
Open an issue or a PR then and log it in the recipe.

Reply:
But the author links to a license already. Do I really need to PR upstream? Can't we just include the file they hyperlink to?

Reply:
The license the author links to requires that a copy of the license be distributed with each copy of the software.

Reply:
Okay, I've made an issue: https://github.com/pratas/xs/issues/3

Reply:
It is now updated. Thanks TMiguelT !

Reply:
updated checksums and file

should this remain version `1.0.0`? The tool itself does not seem to have a version number.

Reply:
I set only a simple tag as 2.0 in the release

Reply:
Technically, you called it `v2`, which would indeed be considered equivalent to `2.0.0.0.0`.

As long as this wasn't merged, we can still change this. As a rule, we don't take packages back though, because it breaks things on the user side, so now would be the time to decide on a versioning scheme (semver perhaps). 

One word of advice - if you expect users, put the version number into the code. There are tools that will help do that (bumpversion, rever, ...). It's just a matter of time until someone runs into problems, and knowing which version they have (`xs --version`) help a lot...

Commonly, things like missing files would be patch level changes. So `1.0.1`.

As per `semver`, the assumption is that `patch level` just fixes things, `minor level` is additions (you added a function), `major` is, a bit counter intuitive, when you remove things. 

The point is that I can say that the range from `[1.12,2.0)` is good for me, i.e. `1.12` brought the latest feature I need, and only `2.0` will remove anything I might need. It's similar (not quire the same) to library versions on linux (SO version - difference there is that the last digit actually gives the backwards compatibility range, a bit more complicated)

Reply:
It not supports the --version
~/git/xs$ ./XS --version
Version: 2.1

Reply:
Ah - I didn't see that. Can you make tag and version match? That makes it easier for us, because the download file name and the version of the tool will be the same. Also means that future updates can be picket up by the bot.